### DISCUSSION TITLE
how to run it using Intel Iris XE graphics card?

### DISCUSSION THREAD

**Discussion by md-shourov-hasan** [lllyasviel/Fooocus#3174] [Q&A] [answered]
I am using always cpu option, but it's tooooo much slow.

**Comment by mashb1t** [comment, marked as answer]
Hey @Shourov5275 , you could try your luck following these instructions: https://github.com/lllyasviel/Fooocus/pull/2120
Please provide your insights if they work for you or not, which would provide further insights in if they should be part of the readme or not.
